### PR TITLE
update desc for map in all examples

### LINKED ISSUE #11797: [examples] add desc to `dataset.map` to improve tqdm bars
Once https://github.com/huggingface/datasets/pull/2374 is merged we should deploy this feature in our examples.

Currently we get a bunch of bars that are absolutely meaningless and hard to understand what they do. See also: https://github.com/huggingface/datasets/issues/2330

The only issue is how to depend on `datasets` dev versions, might have to wait for a new `datasets` release to be able to merge such PR.

### PR DESCRIPTION
**Pull request by bhavitvyamalik**
# What does this PR do?

Closes #11797. I've added the remaining the `desc` for `summarization`, `token-classification,` `translation`, `language-modeling ` and updated `requirements.txt` as well. I wasn't sure what to add in `desc` at some places so I've added `# not sure if it's right` comment there. Please let me know if that description looks good or should I replace it with something else?

## Who can review?
@stas00 @sgugger

### CODE REVIEW

File: examples/pytorch/language-modeling/run_plm.py [hunk 1]
```diff
@@ -402,6 +404,7 @@ def group_texts(examples):
             batched=True,
             num_proc=data_args.preprocessing_num_workers,
             load_from_cache_file=not data_args.overwrite_cache,
+            desc="Running tokenizer on concatenated texts",  # not sure if it's right
```
Comment (sgugger) [inline examples/pytorch/language-modeling/run_plm.py:407]:
```suggestion
            desc=f"Grouping texts in chunks of {max_seq_length}",
```

File: examples/pytorch/language-modeling/run_clm.py [hunk 2]
```diff
@@ -401,6 +404,7 @@ def group_texts(examples):
         batched=True,
         num_proc=data_args.preprocessing_num_workers,
         load_from_cache_file=not data_args.overwrite_cache,
+        desc="Running tokenizer on concatenated texts",  # not sure if it's right
```
Comment (sgugger) [inline examples/pytorch/language-modeling/run_clm.py:407]:
```suggestion
        desc=f"Grouping texts in chunks of {max_seq_length}"
```

File: examples/pytorch/language-modeling/run_clm_no_trainer.py [hunk 3]
```diff
@@ -346,6 +351,7 @@ def group_texts(examples):
         batched=True,
         num_proc=args.preprocessing_num_workers,
         load_from_cache_file=not args.overwrite_cache,
+        desc="Running tokenizer on concatenated texts",  # not sure if it's right
```
Comment (sgugger) [inline examples/pytorch/language-modeling/run_clm_no_trainer.py:354]:
```suggestion
        desc=f"Grouping texts in chunks of {max_seq_length}"
```

File: examples/pytorch/language-modeling/run_mlm.py [hunk 4]
```diff
@@ -424,6 +428,7 @@ def group_texts(examples):
             batched=True,
             num_proc=data_args.preprocessing_num_workers,
             load_from_cache_file=not data_args.overwrite_cache,
+            desc="Running tokenizer on concatenated texts",  # not sure if it's right
```
Comment (sgugger) [inline examples/pytorch/language-modeling/run_mlm.py:431]:
```suggestion
            desc=f"Grouping texts in chunks of {max_seq_length}"
```

**Review by sgugger — APPROVED** [review]
Thanks for adding this to all examples!

File: examples/pytorch/language-modeling/run_clm.py (hunk 2 above)
Comment (bhavitvyamalik) [reply examples/pytorch/language-modeling/run_clm.py:407]: There's no `max_seq_length` variable here :/

File: examples/pytorch/language-modeling/run_clm_no_trainer.py (hunk 3 above)
Comment (bhavitvyamalik) [reply examples/pytorch/language-modeling/run_clm_no_trainer.py:354]: There's no `max_seq_length` variable here too

File: examples/pytorch/language-modeling/run_clm.py (hunk 2 above)
Comment (sgugger) [reply examples/pytorch/language-modeling/run_clm.py:407]: Sorry, it's `block_size` in this example.

File: examples/pytorch/language-modeling/run_clm_no_trainer.py (hunk 3 above)
Comment (sgugger) [reply examples/pytorch/language-modeling/run_clm_no_trainer.py:354]: Same here.

**Review by stas00 — APPROVED** [review]
Thank you for the improvements, @bhavitvyamalik 

Thank you, @sgugger for the overseeing the process!

**Pull request merged**